stubs/docker/docker/context/context.pyi

```diff
@@ -1,47 +1,76 @@
-from _typeshed import Incomplete
+from collections.abc import Mapping
+from typing import Any, TypedDict, type_check_only
+
+@type_check_only
+class StorageData(TypedDict):
```

Comment:
While we recently started to remember the existence of the `@type_check_only` decorator, we still use a leading underscore as a marker for these classes, since auto completion in editors will otherwise often start to suggest these, and editors usually don't understand the decorator.